sources/packages/backend/libs/integrations/src/sfas-integration/sims-to-sfas.processing.service.ts

```diff
@@ -0,0 +1,134 @@
+import { Injectable } from "@nestjs/common";
+import { SIMSToSFASService } from "../services/sfas";
+import {
+  SIMS_TO_SFAS_FILE_NAME_TIMESTAMP_FORMAT,
+  SIMSToSFASProcessingResult,
+  SIMSToSFASStudents,
+} from "./sfas-integration.models";
+import { ConfigService } from "@sims/utilities/config";
+import { ProcessSummary } from "@sims/utilities/logger";
+import { formatDate } from "@sims/utilities";
+import { SIMSToSFASIntegrationService } from ".";
+
+@Injectable()
+export class SIMSToSFASProcessingService {
+  private readonly ftpSendFolder: string;
+  constructor(
+    config: ConfigService,
+    private readonly simsToSFASService: SIMSToSFASService,
+    private readonly simsToSFASIntegrationService: SIMSToSFASIntegrationService,
+  ) {
+    this.ftpSendFolder = config.sfasIntegration.ftpSendFolder;
+  }
+
+  async processSIMSUpdates(
```

Comment:
As it is right now it seems this method will handle the applications and restrictions also. Is it possible to maybe create some private methods to deal with students, applications, and restrictions or it is the expectation that a possible refactor can potentially happen in the upcoming bridge tickets?

Reply:
As discussed, we can leave it to the upcoming bridge tickets to have the private methods implemented.